model/src/main/kotlin/OrtResult.kt

```diff
@@ -95,6 +95,10 @@ data class OrtResult(
         )
     }
 
+    /** An object that can be used to navigate the dependency information contained in this result. */
+    @get:JsonIgnore
```

Comment:
Now this annotation should not be required anymore, or?

Reply:
Yes, it was dropped.

Reply:
[Turn out](https://github.com/oss-review-toolkit/ort/pull/4166#issuecomment-874807742) it was still needed.